### PR TITLE
Allow to copy dot files

### PR DESCRIPTION
**Pull request by ogizanagi**
For instance, when building a static site for GH pages, to include a [`.nojekyll`](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/) file at the root of the directory.

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -36,6 +36,7 @@ public function getConfigTreeBuilder()
                         'src' => '%kernel.project_dir%/public',
                         'dest' => '.',
                         'fail_if_missing' => true,
+                        'ignore_dot_files' => true,
```
Comment (ogizanagi) [inline src/DependencyInjection/Configuration.php:39]: Perhaps this should be false to include by default dotfiles from the public dir?
Or let the developper handle it per-case, should be fine.

Comment (Tom32i) [reply src/DependencyInjection/Configuration.php:39]: The only web-related public dotfile i can think of is the `.htaccess`.

**Review by Tom32i — APPROVED** [review]
Pour info ça te copiait quoi comme dotfile indésirables ?

**Comment by ogizanagi** [comment]
> Pour info ça te copiait quoi comme dotfile indésirables ?

C'est l'inverse. Cf description 😄 : je souhaitais avoir le `.nojekyll` de copié à partir du dossier public afin de l'avoir à la racine de gh-pages.

**Pull request merged**